source/_patterns/02-organisms/00-global/footer.mustache

```diff
@@ -1,14 +1,24 @@
 <footer class="c-footer" role="contentinfo">
 
-  <div class="container l">
-    {{> molecules-footer-links }}
-    {{> molecules-footer-apps }}
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-sm-2"></div>
+      <div class="col-sm-8 c-footer-col">
+        {{> molecules-footer-links }}
+        {{> molecules-footer-apps }}
+      </div>
+      <div class="col-sm-2"></div>
```

Comment:
Is this really necessary? I think bootstrap won't let the other columns flow into the last 2 even if you don't specify this. Same thing for the header actually.